### PR TITLE
Bump versions of aom and dav1d in CI

### PR DESCRIPTION
**Pull request by shssoichiro**
Builds were failing, so this became necessary.

Also deletes the .travis folder, which was unused.

### CODE REVIEW

File: .github/workflows/rav1e.yml [hunk 1]
```diff
@@ -170,8 +170,11 @@ jobs:
           cd ~/.cache/debs
           xargs -a DEBS curl --remote-name-all
           sha256sum -c CHECKSUMS
+      # libdav1d6 depends on libc6 >= 2.32, which is not included in the docker image
       - name: Install packages
         run: |
+          apt update
+          apt --only-upgrade install libc6
```
Comment (barrbrain) [inline .github/workflows/rav1e.yml:177]: We need `sudo` here:
```suggestion
          sudo apt update
          sudo apt --only-upgrade install libc6
```

**Review by barrbrain — APPROVED** [review]
Fetching packages succeeded, so just need to upgrade `libc6`.

**Comment by barrbrain** [comment]
I think we need to move from `ubuntu-latest` to `ubuntu-22.04` to pick up a newer `libc6`. Here there be dragons.

**Comment by shssoichiro** [comment]
Looks like it still didn't like that. I guess the required libc version just isn't in this ubuntu release.

This is why I use Arch.

**Comment by shssoichiro** [comment]
In the interest of fixing CI, I'm going to revert to the previous dav1d version for now, since I _believe_ it was only libaom that was causing CI to fail.

**Comment by barrbrain** [comment]
On the upside, all the current versions are in Ubuntu 22.04. So we could drop the current method and just:
```bash
sudo apt update
sudo apt install nasm libaom-dev libdav1d-dev
```

File: .github/workflows/rav1e.yml [hunk 2]
```diff
@@ -114,54 +114,10 @@ jobs:
...
-        continue-on-error: true
-        id: debs
-        with:
-          path: ~/.cache/debs
-          key: debs-${{ hashFiles('CHECKSUMS') }}
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install nasm libaom-dev libdav1d-dev
       - name: Fetch packages
```
Comment (barrbrain) [inline .github/workflows/rav1e.yml:121]: Need to also remove `Fetch packages` and `Install packages` steps.

**Pull request merged**